pkg/services/accesscontrol/migrator/migrator.go

```diff
@@ -15,6 +15,10 @@ var (
 	batchSize = 1000
 )
 
+const (
+	maxLen = 40
```

Comment:
nit: make this values a constant that can be shared

[reference](https://github.com/grafana/grafana-enterprise/blob/efaaba32a3ad49507d32460f66f56048b464f7aa/src/pkg/extensions/accesscontrol/database/database.go#L29)